pyvtt/vttfile.py

```diff
@@ -137,7 +137,7 @@ def clean_indexes(self):
         for index, item in enumerate(self):
             item.index = index + 1
 
-    def clean_text(self, tags=False, keys=False,
+    def clean_text(self, tags=False, brackets=False, keys=False,
```

Comment:
@jlopex No CC here yet.

I thought about using kwargs here, but the code was not as clear as it is now 😞